<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ContactWidget</class>
 <widget class="QWidget" name="ContactWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>452</width>
    <height>68</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>68</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>68</height>
   </size>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <property name="leftMargin">
    <number>5</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>5</number>
   </property>
   <property name="horizontalSpacing">
    <number>2</number>
   </property>
   <property name="verticalSpacing">
    <number>0</number>
   </property>
   <item row="0" column="0" rowspan="3">
    <layout class="QGridLayout" name="gridLayout_3">
     <property name="horizontalSpacing">
      <number>4</number>
     </property>
     <property name="verticalSpacing">
      <number>0</number>
     </property>
     <item row="1" column="0">
      <widget class="QLabel" name="avatarLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>58</width>
         <height>58</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>58</width>
         <height>58</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="textFormat">
        <enum>Qt::RichText</enum>
       </property>
       <property name="textInteractionFlags">
        <set>Qt::NoTextInteraction</set>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>26</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="0">
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Minimum</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>3</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <layout class="QGridLayout" name="gridLayout">
     <property name="horizontalSpacing">
      <number>0</number>
     </property>
     <property name="verticalSpacing">
      <number>1</number>
     </property>
     <item row="2" column="0">
      <widget class="QLabel" name="lblNetworkStatus">
       <property name="text">
        <string/>
       </property>
       <property name="textFormat">
        <enum>Qt::RichText</enum>
       </property>
       <property name="textInteractionFlags">
        <set>Qt::NoTextInteraction</set>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="lblUserName">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>15</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="textFormat">
        <enum>Qt::RichText</enum>
       </property>
       <property name="textInteractionFlags">
        <set>Qt::NoTextInteraction</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>204</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="3">
    <widget class="ImageButton" name="lblNetworkIcon" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="text" stdset="0">
      <string/>
     </property>
     <property name="iconSize" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
     <property name="flat" stdset="0">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="3">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="ImageButton" name="callButton" native="true">
       <property name="minimumSize">
        <size>
         <width>23</width>
         <height>23</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>23</width>
         <height>23</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Call</string>
       </property>
       <property name="text" stdset="0">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>4</width>
         <height>21</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="2">
      <widget class="ImageButton" name="chatButton" native="true">
       <property name="minimumSize">
        <size>
         <width>23</width>
         <height>23</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>23</width>
         <height>23</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Chat</string>
       </property>
       <property name="text" stdset="0">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>4</width>
         <height>21</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="4">
      <widget class="ImageButton" name="smsButton" native="true">
       <property name="minimumSize">
        <size>
         <width>23</width>
         <height>23</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>23</width>
         <height>23</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Text Message</string>
       </property>
       <property name="text" stdset="0">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="5">
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>4</width>
         <height>21</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="6">
      <widget class="ImageButton" name="emailButton" native="true">
       <property name="minimumSize">
        <size>
         <width>23</width>
         <height>23</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>23</width>
         <height>23</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Email</string>
       </property>
       <property name="text" stdset="0">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="7">
      <spacer name="horizontalSpacer_6">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>4</width>
         <height>21</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="8">
      <widget class="ImageButton" name="sendFileButton" native="true">
       <property name="minimumSize">
        <size>
         <width>23</width>
         <height>23</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>23</width>
         <height>23</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Send File</string>
       </property>
       <property name="text" stdset="0">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="9">
      <spacer name="horizontalSpacer_7">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>4</width>
         <height>21</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="10">
      <widget class="ImageButton" name="faxButton" native="true">
       <property name="minimumSize">
        <size>
         <width>23</width>
         <height>23</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>23</width>
         <height>23</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Send Fax</string>
       </property>
       <property name="text" stdset="0">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="11">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>4</width>
         <height>21</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="12">
      <widget class="ImageButton" name="profileButton" native="true">
       <property name="minimumSize">
        <size>
         <width>23</width>
         <height>23</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>23</width>
         <height>23</height>
        </size>
       </property>
       <property name="toolTip">
        <string>View Profile</string>
       </property>
       <property name="text" stdset="0">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="13">
      <spacer name="horizontalSpacer_8">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>4</width>
         <height>24</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="3">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>11</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ImageButton</class>
   <extends>QWidget</extends>
   <header>qtutil/ImageButton.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../../resources/wengophone.qrc"/>
 </resources>
 <connections/>
</ui>
